app/views/core/CreateAccountModal/teacher/TeacherSignupStoreModule.coffee

```diff
@@ -125,7 +126,7 @@ module.exports = TeacherSignupStoreModule = {
           window.application.tracker?.trackEvent 'Finished Signup', category: "Signup", label: loginMethod
         )
 
-        return Promise.all(trackerCalls)
+        return Promise.all(trackerCalls).catch(->)
```

Comment:
For my own understanding, is this explicitly swallowing a promise error?

Reply:
Yup - added comments here too